docs/sentry/README.md

```diff
@@ -0,0 +1,20 @@
+# Sentry
+
+
+### What is Sentry?
+Sentry is an open-source software platform that specializes in error monitoring and crash reporting for applications.  
+It helps us to track, identify, and resolve issues that occur within our software applications in real-time.  
+By capturing and analyzing errors, exceptions, and crashes, Sentry provides valuable insights into the health and stability of our applications, allowing us to proactively address issues and improve overall software quality.
+
+### What do we track
+As the first iteration of implementing Sentry in our products we only track crashes and no further tracing.
+Crashes can be handled (like in try/catch) and unhandled (like complete server crash).
```

Comment:
Sentry also tracks console logs (log, info, warning, error) by default and therefore kind serves as log rotation tool although log rotation is a whole different thing. But would be good to have this here as well as one might organise logs differently